src/presentations/indexable-post-type-presentation.php

```diff
@@ -50,6 +58,27 @@ public function generate_meta_description() {
 		return $this->options_helper->get( 'metadesc-' . $this->model->object_sub_type );
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	public function generate_og_description() {
+		if ( $this->model->og_description ) {
+			return $this->model->og_description;
+		}
+
+		$og_description = $this->meta->get_value( 'opengraph-description', $this->model->object_id );
```

Comment:
This is exactly the same as above. We fill the og_descripion with the meta value. Thus this line is redundant.

Reply:
Right, that was the whole point 😞